docs/_scripts/feature-selection.py

```diff
@@ -0,0 +1,138 @@
+from pathlib import Path
```

Comment:
Isn't there a markdown file missing for the docs that's supposed to use this script?